tests/test_mvkde.py

```diff
@@ -219,7 +219,8 @@ def test_univariate_two_sample(i):
     ax.grid()
```

Comment:
Both `tests/test_normal_categorical.py` and `test/test_mvknn.py` have a copy-pasta of this code changed here -- does that not need to be rewritten as well?